variables.tf

```diff
@@ -73,3 +73,8 @@ variable "target_revision" {
   type        = string
   default     = "v1.0.0-alpha.1" # x-release-please-version
 }
+
+variable "app_autosync" {
+  type = bool
+  default = false
+}
```

Comment:
Same comment as here: https://github.com/camptocamp/devops-stack-module-traefik/pull/25#discussion_r1044200571

Reply:
After discussion, will leave as is for now. These changes are urgently needed for Scaleway.